js/share.js

```diff
@@ -36,16 +36,14 @@ function share(dest) {
 }
 
 function init_clipboard() {
-	$('#copylink').click(function(){
-		$(this).select();
-	})
-	
-	$('#copylink').zclip({
-		path: zclipurl,
-		copy: $('#copylink').val(),
-		afterCopy:function(){
-			html_pulse( '#copybox h2, #copybox h3', 'Copied!' );
-		}
-	});
-};                     
-
+    var clipboard = new Clipboard('#copylink', {
+        text: function (trigger) {
+            return $(trigger).val();
+        }
+    });
+    
+    clipboard.on('success', function (e) {
```

Comment:
from https://clipboardjs.com/

```
var clipboard = new Clipboard('.btn');

clipboard.on('success', function(e) {
    console.info('Action:', e.action);
    console.info('Text:', e.text);
    console.info('Trigger:', e.trigger);

    e.clearSelection();
});

clipboard.on('error', function(e) {
    console.error('Action:', e.action);
    console.error('Trigger:', e.trigger);
});
```

(e) could contain some info but do you want to log the success?